docs/source-fabric/advanced/model_parallel/fsdp.rst

```diff
@@ -1,12 +1,9 @@
-###########################################
-Training models with billions of parameters
-###########################################
+#####################################################
+Training models with billions of parameters with FSDP
+#####################################################
 
 Use Fully Sharded Data Parallel (FSDP) to train large models with billions of parameters efficiently on multiple GPUs and across multiple machines.
 
-.. note:: This is an experimental feature.
-
-
 Today, large models with billions of parameters are trained with many GPUs across several machines in parallel.
 Even a single H100 GPU with 80 GB of VRAM (the biggest today) is not enough to train just a 30B parameter model (even with batch size 1 and 16-bit precision).
```

Comment:
Since there's the GB200 now:

```suggestion
Even a single H100 GPU with 80 GB of VRAM (one of the biggest today) is not enough to train just a 30B parameter model (even with batch size 1 and 16-bit precision).
```